### PR TITLE
Crash on change graph file

### LINKED ISSUE #216: Crash on change graph file
> **Note**
> This bug was found writing the interface tests.

Testing the application, I came to a part where I wanted to change a graph file. As soon as I changed it, the software crashed.

  - Debug
 ```
Traceback (most recent call last):
  File "C:\Users\ferso\OneDrive\Documentos\Faculdade\GraphFilter\source\controller\wizard_controller.py", line 321, in on_update_graph_file
    self.save_graph_file_path(file_path[0], input_file[-1])
NameError: name 'input_file' is not defined
```
  - Example
  
New-Project-2022-12-18-15-42-14

### PR DESCRIPTION
**Pull request by rosaigor138**
Closes #216

### CODE REVIEW

File: source/controller/wizard_controller.py [hunk 1]
```diff
@@ -370,19 +370,11 @@ def on_button_method_clicked(self):
...
-        self.graph_files_page.tr("Text files (*.g6)")
-        file_path = file_dialog.getOpenFileName(
-            filter="Graph6 Files (*.g6 *.txt *.g6.gz *.txt.gz);;"
-                   "Text files (*.txt *.txt.gz);;"
-                   "Graph6 files (*.g6 *.g6.gz)"
-        )
-
-        self.save_graph_file_path(file_path[0], input_file[-1])
-    """""
+        temp1 = self.graph_files_page.list_files_input.count()
```
Comment (LBeghini) [inline source/controller/wizard_controller.py:374]: What is `temp1`? Is this a descriptive variable name?

**Review by LBeghini — CHANGES_REQUESTED** [review]
Testing the application, it worked perfectly. But the code need some changes.

File: source/controller/wizard_controller.py [hunk 2]
```diff
@@ -392,12 +384,14 @@ def on_insert_graph_file_path(self):
     def save_graph_file_path(self, path, line_input):
         if validate_file(path):
             if path not in wizard_information_store.temp_graph_input_files and path != '':
+                # for item in line_input:
```
Comment (LBeghini) [inline source/controller/wizard_controller.py:387]: I guess this is an unnecessary comment

**Comment by atilaajones** [comment]
You should include the issue number in the description. This way we'll do a review to understand the motivation behind the update.

File: source/controller/wizard_controller.py (hunk 1 above)
Comment (rosaigor138) [reply source/controller/wizard_controller.py:374]: Sorry, the comment and `temp1` are garbage that I ended up bringing back from another solution I tried before

**Review by fsoupimenta — APPROVED** [review]
I tested it on windows and ubuntu, and it worked fine

**Review by atilaajones — APPROVED** [review]
It's working on Windows.

**Review by LBeghini — APPROVED** [review]
Working on macOS Sonoma version 14.1.1 Chip Apple M1

**Pull request merged**